t/fixtures/01-jobs.pl

```diff
@@ -234,7 +239,7 @@
         VERSION   => '13.1',
         MACHINE   => '32bit',
         backend   => 'qemu',
-        jobs_assets => [{asset_id => 1}, {asset_id => 5}],
+        jobs_assets => [{asset_id => 1}, {asset_id => 5},],
```

Comment:
aehm, this is a leftover when I tried to add asset 6 here - but that caused too many unrelated tests to fail, so I removed it again. It won't hurt though